fenix/app/src/main/res/values-night/styles.xml

```diff
@@ -4,8 +4,10 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <style name="SplashScreen" parent="Theme.SplashScreen">
-        <item name="windowSplashScreenBackground">@color/fx_mobile_layer_color_1</item>
-        <item name="windowSplashScreenAnimatedIcon">@drawable/splash_screen</item>
+        <item name="windowSplashScreenBackground">@android:color/transparent</item>
+        <item name="windowSplashScreenAnimatedIcon">@drawable/animated_splash_screen</item>
+        <!-- Splash screen is animated to maximum 12 seconds -->
+        <item name="windowSplashScreenAnimationDuration">12000</item>
```

Comment:
For documentation purposes, where does this number come from? Should it be a dimen, since it's in both style files?

Reply:
Oh that's weird.  Do you have your app in Dark mode? Does it do this all the time?
Unfortunately `windowSplashScreenAnimationDuration` needs to be in the styles.xml because Android it the one reading it, we're not using that value in code.  This value determines when the animation will stop.  From product's decision, we are setting this to 12 seconds because 11 seconds is the maximum possible value that we might need it at.

Reply:
I can't reproduce that locally but I'll investigate to see why.  It shouldn't switch between light and dark in the middle of the animation.

Reply:
Ah, I see what you mean.  Agreed, I'll move it to dimen, thanks

Reply:
Looking into the change in splash screen background colour, there's nothing we can do.  I set the background of the animation to transparent so the background colour is chosen by Android OS.  

I am not sure why it changes colour half way but we have no control over that colour unless we force our own colour which I don't think is a good solution.

Reply:
OK, as long as it's an accepted behavior, I'm cool with approving these changes.